src/app/containers/CpsAsset/index.jsx

```diff
@@ -40,58 +38,72 @@ const SingleContentWrapper = styled.div`
   }
 `;
 
-const FeaturesAnalysis = ({ content }) => {
+const CpsAsset = ({
+  title,
+  content,
+  enableGridWrapper,
+  a11yAttributes,
```

Comment:
since the `a11yAttributes` would mostly contain similar attributes apart from the `aria-labelledby` attrs would it be worth defining the common ones here and just passing a `labelledBy` prop for `aria-labelledby`  from each of the component using it ?

Reply:
How about an `id` prop? Since the `labelledBy` points to an element's id.

Reply:
Updated to use `labelId` prop, since `id` already has an implicit meaning.